### PR TITLE
add color palette to ampvis2 heatmap

### PR DESCRIPTION
**Pull request by MaraBesemer**
FOR CONTRIBUTOR:
* [ ] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/ampvis2/heatmap.xml [hunk 1]
```diff
@@ -150,6 +151,18 @@
                 </param>
             </when>
         </conditional>
+         <param argument="color_palette" type="select" label="Color Palette" help="Choose the color palette for the heatmap.">
```
Comment (bernt-matthias) [inline tools/ampvis2/heatmap.xml:154]: Can you add a test or modify an existing test such that this parameter is covered?

**Comment by bgruening** [comment]
```
Error in amp_heatmap(d, normalise = TRUE, tax_aggregate = "Phylum", tax_add = NULL,  : 
  unused argument (color_palette = c("#000000", "#000000"))
Execution halted

File: tools/ampvis2/heatmap.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool id="ampvis2_heatmap" name="ampvis2 heatmap" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@" license="MIT">
+<tool id="ampvis2_heatmap" name="ampvis2 heatmap" version="@TOOL_VERSION@+galaxy2" profile="@PROFILE@" license="MIT">
```
Comment (bgruening) [inline tools/ampvis2/heatmap.xml:1]:
```suggestion
<tool id="ampvis2_heatmap" name="ampvis2 heatmap" version="@TOOL_VERSION@+galaxy1" profile="@PROFILE@" license="MIT">
```
The current version is `0` https://github.com/galaxyproject/tools-iuc/blob/main/tools/ampvis2/macros.xml#L3

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @MaraBesemer

**Comment by bgruening** [comment]
fixed in the main branch

**Pull request merged**